packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -116,17 +116,22 @@ export default class TableEditor {
             let j = 0;
             for (; j < tr.cells.length; j++) {
                 const td = tr.cells[j];
+                const tableRect = normalizeRect(this.table.getBoundingClientRect());
                 const tdRect = normalizeRect(td.getBoundingClientRect());
 
-                if (!tdRect) {
+                if (!tdRect || !tableRect) {
                     continue;
                 }
 
                 const lessThanBottom = y <= tdRect.bottom;
                 const lessThanRight = this.isRTL ? x >= tdRect.right : x <= tdRect.right;
 
                 if (lessThanRight && lessThanBottom) {
-                    if (i == 0 && y <= tdRect.top + INSERTER_HOVER_OFFSET) {
+                    const isOnTableTop = i === 0;
+                    const isOnLeftOrRight = this.isRTL
```

Comment:
what is this for?

Normally when check rtl case, the condition is in opposite way of ltr case. e.g. if we check "greater than" for LTR:
```
if (a > b) {....}
```

then we check "less than" in rtl:
```if (a < b) {...}```

Not sure why here it checks the same direction in both case. Is this a typo?